perllib/FixMyStreet/Script/UpdateAllReports.pm

```diff
@@ -66,6 +67,21 @@ sub generate {
                 $open{$cobrand}{$body}{$type}++;
             }
         }
+
+        if ( FixMyStreet->config('AREAS_IN_ALLREPORTS_JSON') ) {
+            @areas = grep { $_ } split( /,/, $problem{areas} );
```

Comment:
As we're here, could we drop the broken lines 47-49ish which don't work when body ID != mapit area ID (and even if those do equal, the list pages don't include such reports which is a different issue I guess).